README.md

```diff
@@ -1 +1,274 @@
+        <li><a href="https://git-scm.com/">Git</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>IDE, Desktop Apps, Other Tools</summary>
+      <ul>
+        <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+        <li><a href="https://desktop.github.com/">Github Desktop</a></li>
+        <li><a href="https://www.behance.net/">Behance</a></li>
+        <!-- <li><a href="https://www.figma.com/">Figma</a></li> -->
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+- Mobile First Approach
+- Responsive Website
+- Button Interactions (i.e. hover, etc.)
+- Attractive Images & Design
+- Modal/dialog
+- Dynamic page (data is retrieved from JSON file)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo Link](https://rhk-microverse.github.io/book-awesome-module/)
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+### Hi, there, I'm Rassel - aka [Full Stack Developer] [Check my portfolio](https://rhk-microverse.github.io/My-Portfolio/)
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- git version 2.38.x
+- node.js version > 12.x
+- IDE
+- browser (chrome, firefox, edge, safari)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:fickryiman/awesome-books.git
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  node install
+```
+
+### Usage
+
+To run the project, execute the following command:
+run live server
+
+<!--
+```sh
+  rails server
+```
+-->
+
+### Run tests
+
+To run tests, run the following command:
+Run Github Actions Test
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+### Deployment
+
+This project is deployed at github pages you can clone it here. [Please click to clone](https://github.com/RHK-MICROVERSE/book-awesome-module)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="Rassel Hassan Kadir"></a>
+- Main Author: **Rassel Hassan Kadir**
+> List of the collaborators of this project.
+[Joseph Ddiiro](https://github.com/Ddiiro)
+  
+  [Live Demo Link](https://rhk-microverse.github.io/book-awesome-module/)
+
+👤 **Rassel Hassan Kadir**
+
+<<<<<<< HEAD
+- GitHub: [@githubhandle](https://github.com/RHK-MICROVERSE)
+- Twitter: [@twitterhandle](https://twitter.com/rhk_trading)
+- Linkedin: [@linkedinhandle](https://www.linkedin.com/public-profile/settings?trk=d_flagship3_profile_self_view_public_profile)
+=======
+- GitHub: [@githubhandle](https://github.com/RHK-MICROVERSE/)
+- Twitter: [@twitterhandle](https://twitter.com/rhk_trading/)
+- Linkedin: [@linkedinhandle](https://www.linkedin.com/public-profile/settings?trk=d_flagship3_profile_self_view_public_profile/)
+>>>>>>> 06475ffdea4c768096a503b981e8f6c4e2889f79
+- Email: 9rhktrading@gmail.com
```

Comment:
Please remove these numbers too and the duplicate social media links.

Reply:
well resolved! ✔️